RecoTracker/DeDx/plugins/DeDxEstimatorProducer.cc

```diff
@@ -139,6 +138,7 @@ void DeDxEstimatorProducer::produce(edm::Event& iEvent, const edm::EventSetup& i
         dedxHits.insert(dedxHits.end(), hits.begin(), hits.end());
       }
     } else {
+      dedxHits.reserve(track->recHitsSize() / 2);
```

Comment:
was this supposed to be on L134?
now this line is redundant with L145.
Perhaps remove L145 and keep the original on L133

Reply:
The reason I moved the line inside the if condition is because, when rerunning the module on AOD or miniAOD (where the dedxHitInfo is available and thus I can still rerun it), it fails since the track extras are not available

Reply:
Removed in 5f1453ac892f57775b7ee697ec379f202709097c